assets/js/Line.jsx

```diff
@@ -98,23 +98,21 @@ function Line({
         />
       )}
       {line === 'Silver' && (
-        <div>
-          <label>
-            Bridge Announcements
-            <div className="switch">
-              <input
-                name="chelsea_boarding"
-                type="checkbox"
-                className="switch-input"
-                checked={chelseaBridgeAnnouncements === 'auto'}
-                onChange={(e) => {
-                  setChelseaBridgeAnnouncements(e.target.checked ? 'auto' : 'off');
-                }}
-              />
-              <span className="switch-label">Switch</span>
-            </div>
-          </label>
+      <label className="mt-1 mb-4">
+        Chelsea Drawbridge Announcements
+        <div className="switch">
+          <input
+            name="chelsea_boarding"
```

Comment:
I know this didn't actually change with this PR, but is this `name` actually referenced anywhere? I worry that calling it `chelsea_boarding` instead of `chelsea_bridge` could confuse somebody down the line.

Reply:
it's used in a test. but sure i can change it.